client/overview/task-list/test/tasks.js

```diff
@@ -354,7 +354,7 @@ describe( 'getTasks()', () => {
 					completed: false,
 					level: 1,
 					title: 'Respond to a dispute for $10.00 – Last day',
-					content: 'Respond today by 11:59 PM',
+					content: 'Respond today by 6:59 PM', // shown in local timezone.
```

Comment:
I had a look and couldn't find the local timezone set anywhere. This could cause failures if the timezone is different from the expected (EST ?).

Reply:
Nice catch! I've explicitly set this to `America/New_York` for this test suite, reverting afterAll.

You can check this is working by settings the TZ to e.g. `Australia/Brisbane`, tests will then fail.

Updated in 02cbd09.

Reply:
Set to UTC-5 (EST / non-daylight-savings) in caac016.

Reply:
Shouldn't be affected by daylight savings as the dates are consistent.

Reply:
Ah, yes, right!

I'm having flashbacks of all of the failing GH checks last time we changed to/from daylight savings. I'll just leave it in, at least it's explicit.